products/compute/terraform.yaml

```diff
@@ -233,6 +233,46 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         diff_suppress_func: 'compareSelfLinkOrResourceName'
       timeoutSec: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  RegionBackendService: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_backend_service_basic"
+        primary_resource_id: "default"
+        vars:
+          region_backend_service_name: "region-backend-service"
+          health_check_name: "health-check"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
```

Comment:
I think most of this shouldn't be here w/o `iap` and `security_policy` right?

Reply:
Right! Removing